agent/testflinger_agent/agent.py

```diff
@@ -136,7 +165,13 @@ def process_jobs(self):
                     self.client.config.get("execution_basedir"), job.job_id
                 )
                 os.makedirs(rundir)
+
                 self.client.post_agent_data({"job_id": job.job_id})
+
+                # handle job attachments, if any
+                if job_data.get("attachments", "none") == "complete":
```

Comment:
The `"none"` literal string seems weird, as it's not used for comparison, and `dict.get()` has a default value of `None` (the value, not the string) anyway:

```suggestion
                if job_data.get("attachments") == "complete":
```

Reply:
`"none"` is actually an (admittedly poorly chosen) constant that signifies that this job does not contain attachments. I added that default here so that existing tests that don't involve attachments don't fail. Having said that, you're probably right: we can just assume that the `attachments` field is simply not set when attachments are not present, rather than setting it to `None`.